cost-analyzer/values.yaml

```diff
@@ -517,6 +534,21 @@ ingress:
   #    hosts:
   #      - cost-analyzer.local
 
+apiIngress:
```

Comment:
Let's remove `apiIngress` since it's not used in the template anywhere?

Reply:
That slipped in from how I was doing it before, I'll remove it, thanks!